build.ts

```diff
@@ -144,33 +342,44 @@ async function makeThemeObject(
 
 async function generateVariants(
   name: string,
-  palette: Labels<Color, AlphaColor>
+  flavor: CatppuccinFlavor,
 ) {
   for (const accent of accents) {
-    const theme = await makeThemeObject(name, accent, palette);
+    const theme = await makeThemeObject(name, accent, flavor);
+    const darkLightTheme = await makeDarkLightThemeObject(name, accent, flavor);
+    const fileName = `${name}-${accent}.xpi`;
+    writeTheme(fileName, theme, `./themes/${name}`);
+    writeTheme(fileName, darkLightTheme, `./themes/dark-light/${name}`);
+  }
+}
+
+async function writeTheme(
+  fileName: string,
+  theme: unknown,
```

Comment:
```suggestion
  theme: unknown,
```
Let's make and use a generic `ThemeObject` record type instead of saying this is unknown.

Reply:
I created a ThemeObject to be more type safe.